src/lib/openapi/spec/event-schema.ts

```diff
@@ -0,0 +1,53 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { tagSchema } from './tag-schema';
+
+export const eventSchema = {
+    $id: '#/components/schemas/eventSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['id', 'createdAt', 'type', 'createdBy'],
+    properties: {
+        id: {
+            type: 'integer',
+            minimum: 1,
+        },
+        createdAt: {
+            type: 'string',
+            format: 'date-time',
+        },
+        type: {
+            type: 'string',
+        },
+        createdBy: {
+            type: 'string',
+        },
+        environment: {
+            type: 'string',
+            nullable: true,
+        },
+        project: {
+            type: 'string',
+            nullable: true,
+        },
+        featureName: {
+            type: 'string',
+            nullable: true,
+        },
+        data: {},
+        preData: {},
```

Comment:
Should we use `type: 'object'` here, at least until we have something better?

Reply:
According to the OpenAPI spec, a '{}' means "any type". But sure, we can do a nullable object. That might be better, yeah.

Reply:
Right, I was thinking data/preData would at least be a key/value object (or null, as you say).